### ISSUE TITLE
Issues with maintaining proper context in VS Code Copilot

### ISSUE THREAD

**Issue by tsavo-at-pieces** [pieces-app/support#264]
### Software

VS Code

### Operating System / Platform

macOS

### Your Pieces OS Version

N/A

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

Initial question that came in through our Support Email:

> Why is not remembering the context with which I'm asking a question? I don't want to specify in every question that I'm referring to ruby. But if I don't it gives me answers for all languages that aren't relevant to me. It's frustrating.

Seems like a super valid frustration as it definitely should be picking up on the language. 

@sam-at-pieces @caleb-at-pieces any thoughts here. I'm reaching out for more information!

**Comment by sam-at-pieces** [comment]
Yes this sounds like a particularly annoying issue and one we haven't run into before. Generally, there are a few things we can do to lock a conversation to a specific language(s), either by persisting any detected languages in the copilot memory or injecting them into the prompt as we go. There are some annoying edge cases, however, especially where users are switching between multiple languages or translating from one to another. I've set up an internal meeting to chat over the options

In this particular case it would be super helpful if the user was will to share any or all of:
- the conversation that this was happening in
- the foundation model they were using
- the languages in the conversation and the incorrect responses.
That would help us nail down what went wrong in this particular case.
